src/utils/isDifferentPercentageCovered.js

```diff
@@ -0,0 +1,33 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import formatNumber from "./formatNumber";
+
+const isDifferentPercentageCovered = (populationData, incarcerationRateData) => {
+  if (
+    populationData.countyCoverage !== incarcerationRateData.countyCoverage ||
```

Comment:
I think it would make sense to round these (or format them) before comparing them, so that if they are different but would still display as the same percentages in the frontend we don't show the warning. Otherwise it might be a bit confusing if we show the warning but the coverage numbers displayed are the same.